src/main/java/http/helper/Messages.java

```diff
@@ -0,0 +1,33 @@
+package http.helper;
+
+import java.net.ServerSocket;
+
+public class Messages {
```

Comment:
I like this pattern of encapsulating all of the string messages used throughout the application in a single class.